components/app/manage/project/ProjectOneLiner.vue

```diff
@@ -28,6 +28,7 @@ import AppFormControl from '@/components/common/AppFormControl';
 import AppTextInput from '@/components/common/AppTextInput';
 import ProjectConstants from '@/mixins/ProjectConstants';
 
+/** @deprecated can be replaced with project/TextField */
```

Comment:
question(non-blocking): same question as the other deprecated fields

Reply:
Replaced and removed legacy component